ide/coqide/coqide_ui.ml

```diff
@@ -98,9 +99,11 @@ let init () =
 \n    <menuitem action='Forward' />\
 \n    <menuitem action='Backward' />\
 \n    <menuitem action='Run to cursor' />\
+\n    <menuitem action='Reset' />\
 \n    <menuitem action='Run to end' />\
+\n    <menuitem action='Force' />\
```

Comment:
This doesn't show on the menu,  What is it supposed to do?

Reply:
It is “Fully check” (the ID of this menu item is “Force”) and I don’t know what it is supposed to do actually. As it is present in the toolbar, I wanted to have it as a menu item, at least to show the accelerator.

Reply:
Then it should be "Fully Check" in the menu.  It is briefly described in the [this section](https://coq.inria.fr/distrib/current/refman/practical-tools/coqide.html#asynchronous-mode) of the documentation.

Reply:
“Force” was just an identifier, this is why I didn’t bother changing it. The label was defined in coqide.ml. But OK, this is fixed: the ID is now “Fully check”.